gluejobutils/s3.py

```diff
@@ -47,6 +47,17 @@ def write_json_to_s3(data, s3_path) :
     log_upload_resp = log_obj.put(Body=log_file.getvalue())
     return log_upload_resp
 
+def write_csv_to_s3(data, s3_path) :
+    """
+    Saves your data to a csv file (in memory and then sends it to the s3 path provided)
+    """
+    bucket, key = s3_path_to_bucket_key(s3_path)
+    csv_buffer = StringIO()
+    data.to_csv(csv_buffer)
+    data_obj = s3_resource.Object(bucket, key)
+    data_upload_resp = data_obj.put(Body=csv_buffer.getvalue())
+    return data_upload_resp
+
```

Comment:
Could I recommend that you change the name to:
`write_df_to_csv_on_s3` or something more catchy - basically something that implies that the data is a pandas dataframe? Also change the input var from `data` to `df`

Reply:
changed to `write_df_as_csv_to_s3` for grammatical consistency